### ISSUE TITLE
Notification when opening lid won't show up

### ISSUE THREAD

**Issue by LuiLar** [WillBishop/MenuPower#1]
I gave permissions to the app in Security & Privacy once installed but dunno what else I should do let the app show it's toast notification.

**Comment by WillBishop** [comment]
If you open your AirPods case (with your AirPods) inside near the MacBook, do you get a notification?

**Comment by LuiLar** [comment]
Nope, neither do I get the notification, nor the app starts.

I have to open the app manually every time.

**Comment by WillBishop** [comment]
Yeah the start at login option doesn’t always work. But you should only have to open it once each time you restart.

**Comment by LuiLar** [comment]
Release 1.0.1 does fix the issue, but only is the app is open.

From what I can saw in your Twitter post, when you open the lid, the app starts and notification appears.

Is this feature part of the design of the app?
